.env.development

```diff
@@ -0,0 +1,3 @@
+NEXT_PUBLIC_LVL_V1_CONTRACT_ADDRESS=0x095591311B49C21F117a7C1C25EB027e78aB2AE2
+NEXT_PUBLIC_INFURA_ID=4999cf505829466d89fc48f4880195cc
+NEXT_PUBLIC_NETWORK_ID=0x4
```

Comment:
Do we want these to be exposed? Just checking, probably fine if they are for development.

Reply:
Yep! this is all public blockchain stuffs:

This is the contract address on the specified network, so the dApp knows where to interact with:
NEXT_PUBLIC_LVL_V1_CONTRACT_ADDRESS=0x095591311B49C21F117a7C1C25EB027e78aB2AE2

Infura acts as our dApps ethereum gateway, meaning the dApp asks Infura for information about the ethereum network, and also trusts it to submit transactions to:
NEXT_PUBLIC_INFURA_ID=4999cf505829466d89fc48f4880195cc

This is the network ID - 0x4 is Rinkeby for testing, then on the production instances we can set to mainnet (0x01):
NEXT_PUBLIC_NETWORK_ID=0x4

Reply:
Perfect, thanks!